src/components/ButtonGroup/ButtonGroup.js

```diff
@@ -1,10 +1,25 @@
 import React, { Children } from 'react';
+import PropTypes from 'prop-types';
 import styled, { css } from 'react-emotion';
 
 import Button from '../Button';
 import { childrenPropType } from '../../util/shared-prop-types';
+import { isEqual } from '../../util/fp';
 
-const listStyles = ({ theme }) => css`
+const LEFT = 'left';
+const RIGHT = 'right';
+const ALIGMENT_TYPES = [LEFT, RIGHT];
```

Comment:
I think we now have these somewhere in `src/constants/util`, so we don't have to repeatedly define the same constants.